backend/src/utils/jwt.js

```diff
@@ -0,0 +1,23 @@
+require('dotenv').config();
+import jwt from 'jsonwebtoken';
+
+const LOCAL_JWT_KEY = "a20e136e4925467fca61e3f17fabf8de58d7087f";
+
+export const createTokenFor = (user) => {
+  return jwt.sign(
+    {
+      id: user.id,
+    },
+    process.env.JWT_KEY || LOCAL_JWT_KEY,
+    { algorithm: 'HS256' }
+  );
+};
+
+export const verifyToken = (token) => {
+  const cleanedUpToken = token.startsWith('Bearer')
+    ? token.replace('Bearer ', '')
+    : token;
+  return jwt.verify(cleanedUpToken, process.env.JWT_KEY || LOCAL_JWT_KEY, {
```

Comment:
Maybe try to use different environment files with `dotenv -e FILE`.
Here you have an unnecessary check every time a token gets created or verified.